modules/developers-guide/pages/glossary.adoc

```diff
@@ -12,7 +12,7 @@ actor::
 
 [[g-canister]]
 canister::
-  A canister is a conceptual object with a universally-unique identifier and an owner that defines the boundaries of a specific application, service, or micro-site. 
+  A canister is a conceptual object similar to a smart contract with a universally-unique identifier and an owner that defines the boundaries of a specific application, service, or micro-site. 
```

Comment:
suggest taking this out of the glossary and adding it to the introduction of the canister?